src/hooks/UrqlUseSubscription.rei

```diff
@@ -2,16 +2,9 @@ type handler('acc, 'resp, 'ret) =
   | Handler((option('acc), 'resp) => 'acc): handler('acc, 'resp, 'acc)
   | NoHandler: handler(_, 'resp, 'resp);
 
-type useSubscriptionResponse('ret) = {
-  fetching: bool,
-  data: option('ret),
-  error: option(UrqlCombinedError.t),
-  response: UrqlTypes.response('ret),
-};
-
 let useSubscription:
   (
     ~handler: handler('acc, 'resp, 'ret),
     ~request: UrqlTypes.request('resp)
   ) =>
-  useSubscriptionResponse('ret);
+  UrqlTypes.hookResponse('ret);
```

Comment:
Is there a particular reason we don't `open UrqlTypes` at the top in this file as well? I'm guessing it's because we still have those defs for `handler` hanging around, yeah? I'll remove those once we convert `Subscription` over 👍

Reply:
There was no problem in opening `UrqlTypes` on that file, just changed it! Thanks